meson.build

```diff
@@ -1731,6 +1732,19 @@ install_libudev_static = static_library(
         c_args : static_libudev_pic ? [] : ['-fno-PIC'],
         pic : static_libudev_pic)
 
+systemd_tpm2_plugin = shared_library(
+        'cryptsetup-token-systemd-tpm2',
+        version : systemd_tpm2_plugin_version,
+        link_args : ['-shared',
+                     '-Wl,--version-script=' + systemd_tpm2_plugin_sym_path],
+        dependencies : libshared_deps + [libcryptsetup],
+        link_with : [libshared],
+        link_whole : [systemd_tpm2_plugin_static],
+        link_depends : systemd_tpm2_plugin_sym,
+        install_rpath : rootlibexecdir,
+        install : true,
+        install_dir : rootlibdir)
```

Comment:
is this correct? i.e. does cryptsetup pick up the plugins in the regular (root) libdir? i.e. not a cryptsetup-specific subdir somewhere?

Reply:
yes, currently libcryptsetup searches for token handlers in root libdir